src/Routes.jsx

```diff
@@ -1,40 +1,99 @@
 import React, { Component } from 'react';
-import {
-  graphql,
-  createFragmentContainer,
-} from 'react-relay';
+import DeckGL, { GeoJsonLayer, IconLayer } from 'deck.gl';
 import propTypes from 'prop-types';
+import { createFragmentContainer, graphql } from 'react-relay';
+import muniRoutesGeoJson from './res/muniRoutes.geo.json';
+
+const routesLayer = new GeoJsonLayer({
+  id: 'muni-routes-geojson',
+  data: muniRoutesGeoJson,
+  filled: true,
+  stroked: false,
+  extruded: true,
+});
 
 class Routes extends Component {
+  getMarkers() {
+    /* returns new DeckGL Icon Layer displaying all vehicles and stops on given routes */
+    const ICON_MAPPING = {
+      marker: {
+        x: 0, y: 0, width: 128, height: 128, mask: true,
+      },
+    };
+    const data = [];
+
+    // Push stop markers into data array
+    // because route.stops return null now, control stops before map function
+    this.props.state.routes.map(route => route.stops && route.stops.map(stop =>
+      data.push({
+        position: [stop.lon, stop.lat], icon: 'marker', size: 72, color: [0, 255, 0],
+      })));
+
+    /* Push vehicle markers into data array */
+    this.props.state.routes.map(route => route.vehicles.map(vehicle =>
+      data.push({
+        position: [vehicle.lon, vehicle.lat], icon: 'marker', size: 72, color: [255, 0, 0],
+      })));
+
+    return (new IconLayer({
+      id: 'icon-layer',
+      data,
+      iconAtlas: 'https://uber.github.io/deck.gl/images/icon-atlas.png',
```

Comment:
could we save this image into `./res`?

Reply:
yes, then we can use it like: 
iconAtlas: require('./res/icon-atlas.png');
is it ok?

Reply:
will that syntax work?

Reply:
following works. we may use express server for statics in future.

const atlasIcon = require('./res/icon-atlas.png');
iconAtlas: atlasIcon

Reply:
Ok, now it does. We could, but let's keep things simple for now :)